doc/releases/changelog-dev.md

```diff
@@ -47,6 +47,33 @@
 * New basis sets, `6-311g` and `CC-PVDZ`, are added to the qchem basis set repo.
   [#3279](https://github.com/PennyLaneAI/pennylane/pull/3279)
 
+* We can now perform QPE on quantum operations.
```

Comment:
```suggestion
* QPE can now accept a target operator instead of a matrix and target wires pair.
  [(#3373)](https://github.com/PennyLaneAI/pennylane/pull/3373)
```
This entry should also go under "Improvements" instead of "New Features".